packages/flutter/test/scheduler/benchmarks_test.dart

```diff
@@ -64,5 +68,5 @@ Future<void> main() async {
       expect(endFramesBegun, equals(startFramesBegun + 1));
       expect(endFramesDrawn, equals(startFramesDrawn + 1));
     });
-  });
+  }, tags: 'web_unimplemented');
```

Comment:
i'd prefer if we kept the number of `test` features we are using here to a minimum.

Reply:
Is there a good alternative in the set of features we are already using? `tags` seems like the most idiomatic way to do this (they were added for this exact use-case; I added this feature :sweat_smile: to `package:test`).

Reply:
`skip`?

Reply:
My understanding is that `skip` always skips. You can't control it from the command-line. So, for example, if you fix zero or more tests that are marked with `web_unimplemented`, you still have the option to run them by passing the appropriate command-line args.

Reply:
I though about that to, but in this case if you are using `skip: isBrowser` you can just change that value to false and re-run

Reply:
or comment out the line. why would you need to control it from the command line?